### ISSUE TITLE
I am Facing Undefined symbol: __swift_FORCE_LOAD_$_swiftUniformTypeIdentifiers Undefined symbol: __swift_FORCE_LOAD_$_swiftCoreMIDI Undefined symbol: __swift_FORCE_LOAD_$_swiftWebKit while tring to build project from xcode

### ISSUE THREAD

**Issue by chetan-sharma-newput** [react-native-community/upgrade-support#129]
## Things I’ve done to figure out my issue
I have done all the solutions provided by google but no luck. I created a swift file. but while creating a swift file I noticed Xcode is not asking me to create a bridging header, it directly creates the swift file. I thought this may be the issue. but again no luck. So help me out I am upgrading my project 0.61.x to 0.63.3.

- [ ] I used [upgrade-helper](https://react-native-community.github.io/upgrade-helper/) to do my upgrade.

## Description

I am facing issues while trying to build my project with Xcode 12.2. I upgraded my project's react-native version from 0.61 to 0.63. Now I am unable to build my project and facing these issues. 

 **Undefined symbol: __swift_FORCE_LOAD_$_swiftUniformTypeIdentifiers
 Undefined symbol: __swift_FORCE_LOAD_$_swiftCoreMIDI
 Undefined symbol: __swift_FORCE_LOAD_$_swiftWebKit** 

![Screenshot 2020-12-15 at 1 32 44 PM](https://user-images.githubusercontent.com/59158878/102196691-3f47c380-3ee6-11eb-9a2e-fa3379410546.png)

**Comment by iamabdulhaseeb** [comment]
@chetan-sharma-newput Did you solve it?

**Comment by chetan-sharma-newput** [comment]
Yeah, In my case I was having two .swift files when I deleted both (the old one and the new one) and created only a new one (File.swift). My problem gets solved.

**Comment by chetan-sharma-newput** [comment]
> @chetan-sharma-newput Did you solve it?
Yeah, In my case I was having two .swift files when I deleted both (the old one and the new one) and created only a new one (File.swift). My problem gets solved.

**Comment by eramudeep** [comment]
> > @chetan-sharma-newput Did you solve it?
> > Yeah, In my case I was having two .swift files when I deleted both (the old one and the new one) and created only a new one (File.swift). My problem gets solved.

Yes, this worked for me as well

**Comment by musabgulfam** [comment]
Facing the same issue, any solution?

**Comment by chetan-sharma-newput** [comment]
> Facing the same issue, any solution?

try this, I already shared what I did.
https://github.com/react-native-community/upgrade-support/issues/129#issuecomment-773066981

**Comment by felipecastrosales** [comment]
any suggestions? I'm facing with this

**Comment by vikas-finity** [comment]
@chetan-sharma-newput where are these files located?

**Comment by oapmartins** [comment]
Any solutions?
